workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+
+![nCOVID table open with inspect element](https://cloud-p5pdve0kf.vercel.app/1inspect_element.png)
+
+```python
+# get world data table
+table = soup.find('table', attrs={'id':'sortable_table_world'})
+```
+After this code runs we now have a new beautiful soup object called `table` that contains the table and child elements within that table.
+
+With `table` we will use the `findAll` function to get all the `tr` tags from the table. `tableRows` is a Python list with a beautiful soup object for each `tr` tag found in the table.
+```python
+# get table rows
+tableRows = table.findAll('tr')
+```
+Now that we have all the rows in a Python list, we can iterate through the list and try to find the United States or you can input any Country you want.
+
+Before getting to the looping part though we will need to take a look at some of our outputs. This may happen to you when webscraping, the text has weird formatting. In these next steps I'm going to show the text from the website with the weird formatting and how we can use Python to remove that.
+
+I will just use the second row and first column for this example
+```python
+tableCols = tableRows[1].findAll('td')
+tableCol = tableCols[0]
+tableCol.text
+```
+This returns `'\r\n          TOTAL\r\n        '` with weird spacing and newline and indents. With Python we can use the `strip` function on the string and we will end with this code `tableCol.text.strip()` and this text `'TOTAL'`. Now that we got that handled lets get to reading the data!
+
+First we have a have a for each loop to loop through every row in the table. You make notice the `[1:]`, this was added to make the loop just a little more effcient because the first row at index `[0]` is just the table header with no data. Next we use an if statment to check if the current row's country is the United States (This can be replaced by any country). **Do Note** that we use the `.find()` function because we are only checking the first column and `.text[2:]` because the website has a weird astricks we need to remove. Once we find the country of our choice, we use `findAll()` to get all the columns in the row and print out some data.
+```python
+for row in tableRows[1:]:
+    if 'United States' == row.find('td').text[2:].strip():
+        columns = row.findAll('td')
+        print(columns[0].text[2:].strip())
+        print(f'Confirmed Cases: {columns[1].text.strip()}')
+        print(f'New Cases: {columns[3].text.strip()}')
+        print(f'Critical Cases: {columns[5].text.strip()}')
+```
+The output at the time of writing this workshop is:
+```
+United States
+Confirmed Cases: 9,566,690
+New Cases: 28,845
+Critical Cases: 88,127
+```
+Scary Numbers! Hopefully at the time of reading this workshop things won't be too much worse :(
+
+Here is some example code for checking for multiple countries
```

Comment:
this confuses me because it's not worded as an instruction, so i'm not sure if i should copy/paste this code in order to continue the workshop or what exactly i should be doing